server/views/referrals/reason.njk

```diff
@@ -0,0 +1,59 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% extends "../partials/layout.njk" %}
+
+{% block personBanner %}
+  {% include "./_personBanner.njk" %}
+{% endblock personBanner %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: referPaths.show({ referralId: referral.id })
+  }) }}
+{% endblock backLink %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% set detailsHtml %}
+      <ul class="govuk-list govuk-list--bullet">
+        <li>you can provide information about why you're referring this person to this programme</li>
+        <li>you can add supporting information that would be useful for the programme team to know</li>
+        <li>you can add any other information that would be useful for the programme team to know</li>
+      </ul>
+      {% endset %}
+
+      {{ govukDetails({
+        summaryText: "What type of information can I add?",
+        html: detailsHtml,
+        attributes: {
+          "data-testid": "information-type-details"
+        }
+      }) }}
+
+      <form action="{{ referPaths.update({ referralId: referral.id }) }}?_method=PUT" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+        {{ govukTextarea({
+          name: "reason",
+          id: "reason",
+          label: {
+            text: "Referral reason and supporting information"
+          },
```

Comment:
This isn't in the design, but I think we do need a visible label of some kind, so it makes sense to include it here. I wonder if it would be better to follow the name in the heading here to make it read a bit softer: "Reason for referral and any additional information"

Reply:
Should the page heading be updated to "supporting" if we're going for this?

Reply:
@yndajas we've had a quick chat about this and have made the label a page title and removed this extra one. I was actually looking at outdated content so it should actually be "additional information"

Reply:
Figma! 😅